blueman/DeviceClass.py

```diff
@@ -53,24 +53,24 @@
     # translators: device class
     _("Modem"),
     # translators: device class
-    _("Isdn")
+    _("ISDN")
 )
 
 access_point_minor_cls = (
     # translators: device class
     _("Fully"),
     # translators: device class
-    _("1-17 percent"),
```

Comment:
These values are probably not very good in general as they lack context. We should probably adapt the ones from https://btprodspecificationrefs.blob.core.windows.net/assigned-numbers/Assigned%20Number%20Types/Baseband.pdf, i.e. include "utilized" and "available".

Reply:
I don't understand what is being utilized or alternatively remains available with the context. It says "Minor Device Class field – LAN/Network Access Point Major Class"

Is it available clients, signal strength, battery level, bandwidth, … ?

Reply:
I don't know either. Probably clients or upstream bandwidth. :shrug: